osgbuild/promoter.py

```diff
@@ -86,6 +86,8 @@ def __init__(self, pkg_or_build, dist, reason):
     def __str__(self):
         return self.reason % {'pkg_or_build': self.pkg_or_build, 'dist': self.dist}
 
+    __repr__ = __str__
```

Comment:
Just curious, is this `__repr__` used for error messages or smth?

Reply:
__repr__ was used when I was printing out lists of these for debugging purposes.